### PR TITLE
Added in URL Command Name Resolver

### PR DESCRIPTION
**Pull request by JoeStead**
* Split integration tests for Header/URL tests (This could be done better, there's a fair amount of duplication).
* Got rid of all of your async void code.

### CODE REVIEW

File: src/Sequin.Autofac/packages.config [hunk 1]
```diff
@@ -1,4 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Autofac" version="3.3.1" targetFramework="net45" />
+  <package id="Autofac" version="3.5.2" targetFramework="net45" />
+  <package id="FluentValidation" version="6.0.2.0" targetFramework="net45" />
```
Comment (jasonmitchell) [inline src/Sequin.Autofac/packages.config]: Sequin.Autofac shouldn't take a dependency on FluentValidation.

File: src/Sequin.ClaimsAuthentication/packages.config [hunk 2]
```diff
@@ -1,5 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
+  <package id="Autofac" version="3.5.2" targetFramework="net45" />
+  <package id="FluentValidation" version="6.0.2.0" targetFramework="net45" />
```
Comment (jasonmitchell) [inline src/Sequin.ClaimsAuthentication/packages.config]: Sequin.ClaimsAuthentication shouldn't take a dependency on Autofac or FluentValidation

File: src/Sequin.Core/packages.config [hunk 3]
```diff
@@ -1,4 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
+  <package id="Autofac" version="3.5.2" targetFramework="net45" />
+  <package id="FluentValidation" version="6.0.2.0" targetFramework="net45" />
```
Comment (jasonmitchell) [inline src/Sequin.Core/packages.config]: Sequin.Core shouldn't take a dependency on Autofac or FluentValidation

File: src/Sequin.FluentValidation/packages.config [hunk 4]
```diff
@@ -1,6 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="FluentValidation" version="5.0.0.1" targetFramework="net45" />
+  <package id="Autofac" version="3.5.2" targetFramework="net45" />
```
Comment (jasonmitchell) [inline src/Sequin.FluentValidation/packages.config]: Sequin.FluentValidation shouldn't take a dependency on Autofac

Comment (JoeStead) [inline src/Sequin.FluentValidation/packages.config]: The last time I trust VS to update dependencies on its own...

**Comment by jasonmitchell** [comment]
Would you be able to remove the commits 01fce787863dc410e51293db8c787dd748cdbe54 and 4f6ab71c72dff40099be36e923e458a20600cece?

It would help to keep the history clean rather than having a "revert" commit in the history

**Comment by JoeStead** [comment]
@jasonmitchell what commits? ;)

**Comment by jasonmitchell** [comment]
I see what you did there!

**Comment by jasonmitchell** [comment]
Closes #5

**Pull request merged**